docker/jar/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM ubuntu
+
+# use a common app path, copied from python-onbuild:latest
+ENV WORKDIR /usr/src/app
+RUN mkdir -p ${WORKDIR}
+WORKDIR ${WORKDIR}
+
+# install dependencies
+RUN apt-get update \
+    && apt-get install -y \
+        curl \
+    && curl -sL https://deb.nodesource.com/setup_6.x -o nodesource_setup.sh \
+    && bash nodesource_setup.sh \
+    && apt-get update \
+    && apt-get install -y \
+        build-essential \
+        git \
+        maven \
+        nodejs \
+        openjdk-8-jdk \
+    && mvn --version \
+    && npm install -g bower
```

Comment:
You do not need to install bower unless you want to recompile the UI (which is not required unless you've made changes to it).